Gemfile.lock

```diff
@@ -361,22 +358,19 @@ GEM
       xcpretty (~> 0.2, >= 0.0.7)
 
 PLATFORMS
-  arm64-darwin-21
-  arm64-darwin-22
-  arm64-darwin-23
-  x86_64-darwin-22
-  x86_64-linux
+  arm64-darwin-24
+  ruby
```

Comment:
I do think this might break CI, we should run all tests. You can do that by commenting "@RCGitBot please test" in this PR